civismlext/test/test_preprocessing.py

```diff
@@ -135,7 +133,7 @@ def levels_dict_numeric():
 def test_sklearn_api():
     name = DataFrameETL.__name__
     check_parameters_default_constructible(name, DataFrameETL)
-    check_no_fit_attributes_set_in_init(name, DataFrameETL)
+    # check_no_attributes_set_in_init(name, DataFrameETL)
```

Comment:
It was a mistake; I meant to remove this entirely. This specific check does not exist in sklearn>=0.20. But it's not really necessary to keep the check, since it's only checking that init follows the convention of not setting attributes that end in `_` during init. This isn't actually a requirement as far as I know, which is probably why it was deprecated. See the source for sklearn 0.19.X https://github.com/scikit-learn/scikit-learn/blob/f0ab589f1541b1ca4570177d93fd7979613497e3/sklearn/utils/estimator_checks.py#L1481